2021-04/cql/GKE/k8ssandra-perf.yaml

```diff
@@ -0,0 +1,55 @@
+cassandra:
```

Comment:
Maybe this could be shared too. The only values that I expect to vary depending on the cloud vendor are `storageClass`, and the `topology.kubernetes.io/zone` label.